Description: Tweak enum for python 3.11 compatibility
 Backport upstream patch for compatibility with python 3.11
Origin: https://github.com/boostorg/python/commit/a218babc8daee904a83f550fb66e5cb3f1cb3013
Acked-By: Anton Gladky <gladk@debian.org>
Last-Update: 2022-12-17
Bug-Debian: https://bugs.debian.org/1025658

--- a/libs/python/src/object/enum.cpp
+++ b/libs/python/src/object/enum.cpp
@@ -119,7 +119,6 @@
 #if PY_VERSION_HEX < 0x03000000
     | Py_TPFLAGS_CHECKTYPES
 #endif
-    | Py_TPFLAGS_HAVE_GC
     | Py_TPFLAGS_BASETYPE,                  /* tp_flags */
     0,                                      /* tp_doc */
     0,                                      /* tp_traverse */
